streamx-console/streamx-console-service/src/main/java/com/streamxhub/streamx/console/core/task/FlinkTrackingTask.java

```diff
@@ -510,7 +510,7 @@ private void getFromYarnRestApi(Application application, StopFrom stopFrom) thro
                     cleanOptioning(optionState, application.getId());
                     this.persistentAndClean(application);
 
-                    if (flinkAppState.equals(FlinkAppState.FAILED) || flinkAppState.equals(FlinkAppState.LOST)) {
+                    if (flinkAppState.equals(FlinkAppState.FAILED) || flinkAppState.equals(FlinkAppState.LOST) || flinkAppState.equals(FlinkAppState.CANCELED)) {
```

Comment:
This will cause all normal cancelled jobs to send alarm notifications ,We can discuss about this

Reply:
Tasks are stopped by user clicks and can be divided into two categories:
1. The action is triggered by the task creator. Then, it is not recommended to send a message notification, such operations are performed independently by the task owner.
2. By a non-task creator, such as the admin user or an administrator in the user group. Then this type of operation can send a message notification, the purpose of which notifies the task creator

Reply:
> Tasks are stopped by user clicks and can be divided into two categories:
> 
> 1. The action is triggered by the task creator. Then, it is not recommended to send a message notification, such operations are performed independently by the task owner.
> 2. By a non-task creator, such as the admin user or an administrator in the user group. Then this type of operation can send a message notification, the purpose of which notifies the task creator

I agree with this approach. In addition, I think the following situations also need to trigger the alarm mechanism:
The action of job cancellation is initiated by an external system (such as Flink Web UI, command line, etc.). Because the job cancellation action initiated by the external system has broken away from the original user system of streamx, I think it should also be Treat it as an exception.

Reply:
+1，Your pr I understand is an alarming strategy, suggest adding an optional configuration item in the UI layer and the back-end system layer